main/views/main.py

```diff
@@ -225,13 +226,27 @@ def get(self, request, *args, **kwargs):
         map_poly = geojson.loads(s)
         entryPolyDict = {}
         entryPolyDict[m_uuid] = map_poly.coordinates
+        for a in Address.objects.filter(entry=user_map):
+            street = a.street
+            city = (a.city + ", " + a.state + " " + a.zipcode)
 
         context = {
+            "street": street,
+            "city": city,
             "c": user_map,
             "entries": json.dumps(entryPolyDict),
             "mapbox_key": os.environ.get("DISTR_MAPBOX_KEY"),
             "mapbox_user_name": os.environ.get("MAPBOX_USER_NAME"),
         }
+        if self.request.user.is_authenticated:
+            if user_map.organization:
+                context["is_org_admin"] = self.request.user.is_org_admin(
+                    Organization.objects.get(slug=user_map.organization.slug).id
```

Comment:
This can be changed to user_map.organization__id I believe - no need to query again!

Reply:
Good point!